metrics/metrics.go

```diff
@@ -0,0 +1,66 @@
+package metrics
+
+import (
+	"encoding/json"
+	"io"
+	"strings"
+
+	metrics "github.com/rcrowley/go-metrics"
+)
+
+const (
+	GRPC = "grpc"
+	SSE  = "sse"
+)
+
+type Metrics struct {
+	Type     string
+	registry metrics.Registry
+}
+
+const (
+	clientCounterName = "clients"
+)
+
+var registry metrics.Registry
+
+func init() {
+	registry = metrics.NewRegistry()
+}
+
+type Registry metrics.Registry
```

Comment:
__[golint]__
* [golint] exported type Registry should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->